src/oauth2-mock-server.ts

```diff
@@ -125,47 +126,44 @@ function parsePort(portStr) {
   return port;
 }
 
-function parseJWK(filename) {
+async function parseJWK(filename: string): Promise<JWK.Key> {
   const jwkStr = fs.readFileSync(filename, 'utf8');
```

Comment:
If we are async now, then this call should also be (via `promisify`, I guess).

Reply:
Fixed